README.md

```diff
@@ -514,7 +514,8 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. 
+    Use keyword arguments (if available - in Ruby 2.0 or later) or an options
```

Comment:
i think it's actually the other way around - default keyword args were added in `2.0`, but REQUIRED keyword args were added in `2.1`